src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -0,0 +1,468 @@
+                    >Terminate</BulkActionButton
+                  >
+                {/if}
+              </div>
+            </th>
+          {:else}
+            {#each pinnedColumns as column}
+              <WorkflowsSummaryTableHeaderCell {column} {sortDisabled} />
+            {/each}
+          {/if}
+        </tr>
+        {#if $updating}
+          <ProgressBar />
+        {/if}
+      </thead>
+      <tbody>
+        {#if $workflowTableColumns.length > 0}
+          {#each workflows as workflow}
+            <tr
+              class="workflow-summary-configurable-row pinned"
+              on:click={() => goToWorkflow(workflow)}
+            >
+              {#if $supportsBulkActions}
+                <td>
+                  <Checkbox
+                    hoverable
+                    bind:group={selectedWorkflows}
+                    value={workflow}
+                    disabled={allSelected}
+                  />
+                </td>
+              {/if}
+              {#each pinnedColumns as column}
+                <WorkflowsSummaryTableBodyCell {column} {workflow} />
+              {/each}
+            </tr>
+          {/each}
+        {:else}
+          <tr />
+        {/if}
+      </tbody>
+    </table>
+  </div>
+  <div class="workflow-summary-table-wrapper">
+    <table class="workflow-summary-table">
+      <thead>
+        <tr class="workflow-summary-header-row">
+          {#each otherColumns as column}
+            <WorkflowsSummaryTableHeaderCell {column} {sortDisabled} />
+          {/each}
+          <th
+            class="h-10 leading-[48px] text-right sticky right-0 block w-auto bg-primary"
+          >
+            <IconButton
+              icon="vertical-ellipsis"
+              on:click={openCustomizationDrawer}
+            />
+          </th>
+        </tr>
+        {#if $updating}
+          <ProgressBar />
+        {/if}
+      </thead>
+      <tbody>
+        {#if $workflowTableColumns.length > 0}
+          {#each workflows as workflow}
+            <tr
+              class="workflow-summary-configurable-row"
+              on:click={() => goToWorkflow(workflow)}
+            >
+              {#each otherColumns as column}
+                <WorkflowsSummaryTableBodyCell {column} {workflow} />
+              {/each}
+              <td />
+            </tr>
+          {:else}
+            <tr>
+              <td colspan={otherColumns.length + 1}>
+                {#if $loading}
+                  <Loading />
+                {:else}
+                  <EmptyState
+                    title="No Workflows Found"
+                    content="If you have filters applied, try adjusting them. Otherwise please check your syntax and try again."
+                    error={$workflowError}
+                  />
+                {/if}
+              </td>
+            </tr>
+          {/each}
+        {:else}
+          <tr>
+            <td>
+              <EmptyState title="No column headers are in view">
```

Comment:
Maybe we want some padding here so the text isn't so close to the edge? 

![Screenshot 2023-03-29 at 3 11 53 PM](https://user-images.githubusercontent.com/15069288/228670322-daaa5c58-2afa-49c4-be63-2bad711a255e.png)